ccd-definition/CaseEvent/User/UserEventsLRspec-GS-prod.json

```diff
@@ -47,7 +47,7 @@
     "PostConditionState": "*",
     "SecurityClassification": "Public",
     "CallBackURLAboutToStartEvent": "${CCD_DEF_CASE_SERVICE_BASE_URL}/cases/callbacks/version/V_1/about-to-start",
-    "CallBackURLAboutToSubmitEvent": "${CCD_DEF_CASE_SERVICE_BASE_URL}/cases/callbacks/about-to-submit",
+    "CallBackURLAboutToSubmitEvent": "${CCD_DEF_CASE_SERVICE_BASE_URL}/cases/callbacks/version/V_1/about-to-submit",
```

Comment:
There should be no prod and non files left after the release, Everything should be copied into the base files. This file is identical to UserEventsLRspec-nonprod.json. Copy the 3 events of this file into UserEventsLRspec.json and delete this file and UserEventsLRspec-nonprod.json